Sources/SwiftParse/TopLevel.swift

```diff
@@ -0,0 +1,14 @@
+import SwiftAST
+import Runes
+import TryParsec
+import Result
+
+let topLevel: SwiftParser<[Statement]> = stmtBraceItems <* OWS <* endOfInput()
+
+public func parse(_ sourceCode: String) throws -> [Node] {
```

Comment:
Hmm. I though top level nodes can contain something other than a statement. However top level nodes must be statements. I'll revert it to return `[Statement]`.